docs/deployment/resource-providers/standalone/kubernetes.md

```diff
@@ -23,7 +23,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-This page describes how to deploy a *Flink Job* and *Session cluster* on [Kubernetes](https://kubernetes.io).
+This page describes how to deploy a *Flink application cluster* and *Session cluster* on [Kubernetes](https://kubernetes.io).
```

Comment:
Couldn't we refactor the documentation here as well sticking to the sections we used for all the other resource provider pages? I have the feeling that it would work here as well 🤔

Reply:
Okay, I'll give it a try